tsuru/client/apps.go

```diff
@@ -847,6 +855,14 @@ func buildScheduleInfo(schedule tsuru.AutoScaleSchedule) string {
 	)
 }
 
+func buildPrometheusInfo(prometheus tsuru.AutoScalePrometheus) string {
+	thresholdValue := strconv.FormatFloat(prometheus.Threshold, 'f', -1, 64)
+
+	return fmt.Sprintf("Name: %s\nQuery: %s\nThreshold: %s\nPrometheusAddress: %s",
```

Comment:
@crgisch prometheusAddress may be blank, I prefer to hide it when is empty

Reply:
@wpjunior prometheusAddress will always have and address. When the user doesnt set a custom address we will return the default one build by the template